src/utils/display-text/format.ts

```diff
@@ -15,6 +15,8 @@ export function weiToNumberFormat(value: any, decimals: number = 0) {
 }
 
 export function toNumberFormat(value: number, decimals: number = 0) {
+  if (isNaN(value)) return '0';
+
```

Comment:
Now it changed what is being returned (no decimals and format), maybe just reset value to 0:

```suggestion
  if (isNaN(value)) value = 0;

```